packages/hadron-build/lib/target.js

```diff
@@ -348,28 +361,23 @@ class Target {
     this.assets = [
       {
         name: `${this.id}-${this.version}-${this.platform}-${this.arch}.exe`,
-        path: this.dest(this.windows_setup_label)
+        path: this.dest(this.windows_setup_label),
+        downloadCenter: true
```

Comment:
We don't list all the assets in the download center, so this explicitly marks the ones that will be shown there